test/dlc_tests/conftest.py

```diff
@@ -54,6 +54,9 @@ def pytest_addoption(parser):
     parser.addoption(
         "--generate-coverage-doc", action="store_true", default=False, help="Generate a test coverage doc",
     )
+    parser.addoption(
+        "--eks-multi-node", action="store_true", default=False, help="Run EKS multi-node tests",
```

Comment:
Can you make this a general "--multinode" flag? We may have multinode ECS tests in the future, for example. You can explicitly run the eks multinode tests via testrunner.